### DISCUSSION TITLE
[Combobox] should never lose focus

### DISCUSSION THREAD

**Discussion by matijs** [tailwindlabs/headlessui#1984] [Ideas]
When selecting a `Combobox.Option` using a keyboard focus never leaves the `Combobox.Input`. When selecting a `Combobox.Option` using a pointer focus is lost (returned to the `<body>` probably).

Like a native `<select>`, it would be nice if focus was returned to `Combobox.Input` upon selection using a pointing device.

**Comment by AnnaYuS** [comment]
Noticed this one as well. If it's not intended behaviour then maybe it should be reported as an issue?

**Reply by matijs** [reply]
My assumption is that the behaviour ought to be the same whether the user uses a keyboard or a pointing device but it would be nice to have this confirmed :)
